snap-core/src/main/java/org/esa/snap/core/datamodel/RGBImageProfile.java

```diff
@@ -425,11 +425,13 @@ public static void storeRgbaExpressions(final Product product, final String[] rg
     public static void storeRgbaExpressions(final Product product, final String[] rgbaExpressions, final String[] bandNames) {
         for (int i = 0; i < bandNames.length; i++) {
             final String rgbBandName = bandNames[i];
-            final String rgbaExpression = rgbaExpressions[i];
+            String rgbaExpression = rgbaExpressions[i];
             final Band rgbBand = product.getBand(rgbBandName);
             final boolean expressionIsEmpty = rgbaExpression.equals("");
```

Comment:
Suggestion:
final boolean expressionIsEmpty = rgbaExpression == null || rgbaExpression.trim().isEmpty();

Reply:
Can't remember any more the reason why we stumbled upon this issue. 
Just asking, because I would like to create an issue. Would be good to know the reason for the description of the error.